cli.php

```diff
@@ -29,6 +29,68 @@ public function __invoke() {
 
 	}
 
+	/**
+	 * Debug object cache hit / miss ratio for any page URL.
+	 *
+	 * ## OPTIONS
+	 *
+	 * [--url=<url>]
+	 * : Execute a request against a specified URL. Defaults to home_url( '/' ).
+	 *
+	 * [--format=<format>]
+	 * : Render the results in a particular format.
+	 *
+	 * @when before_wp_load
+	 */
+	public function debug( $_, $assoc_args ) {
+		global $wp_object_cache;
+		$this->load_wordpress_with_template();
+		$data = array(
+			'cache_hits'      => $wp_object_cache->cache_hits,
+			'cache_misses'    => $wp_object_cache->cache_misses,
+			'redis_calls'     => $wp_object_cache->redis_calls,
+		);
+		WP_CLI::print_value( $data, $assoc_args );
+	}
+
+	/**
+	 * Runs through the entirety of the WP bootstrap process
+	 */
+	private function load_wordpress_with_template() {
+		global $wp_query;
+
+		WP_CLI::get_runner()->load_wordpress();
+
+		// Set up the main WordPress query.
+		wp();
+
+		$interpreted = array();
+		foreach( $wp_query as $key => $value ) {
+			if ( 0 === stripos( $key, 'is_' ) && $value ) {
+				$interpreted[] = $key;
+			}
+		}
+		WP_CLI::debug( 'Main WP_Query: ' . implode( ', ', $interpreted ), 'redis-debug' );
+
+		define( 'WP_USE_THEMES', true );
+
+		add_filter( 'template_include', function( $template ) {
+			$display_template = str_replace( dirname( get_template_directory() ) . '/', '', $template );
+			WP_CLI::debug( "Theme template: {$display_template}", 'redis-debug' );
+			return $template;
+		}, 999 );
+
+		// Template is normally loaded in global scope, so we need to replicate
+		foreach( $GLOBALS as $key => $value ) {
+			global $$key;
+		}
+
+		// Load the theme template.
+		ob_start();
+		require_once( ABSPATH . WPINC . '/template-loader.php' );
+		ob_get_clean();
+	}
+
 }
 
-WP_CLI::add_command( 'redis-cli', 'WP_Redis_CLI_Command' );
+WP_CLI::add_command( 'redis', 'WP_Redis_CLI_Command' );
```

Comment:
> As a point of clarity, if this change were happening after 1.0.0, we'd keep redis-cli and mark it as deprecated, yes?

Yes, although I'm not worried about the semantics in this case. `wp redis-cli` is an alias for launching `redis-cli` with the configuration details in `wp-config.php`. Changing the command won't break any systems, it will just make human operators use something else. And, given this command was first included in the last release, I doubt many people have even discovered it.